docs/src/calling-contracts/call-params.md

```diff
@@ -0,0 +1,39 @@
+# Call parameters
+
+Call parameters are:
+
+1. Amount;
+2. Asset ID;
+3. Gas forwarded.
+
+You can use these to forward coins to a contract. You can configure these parameters by creating an instance of [`CallParameters`](https://github.com/FuelLabs/fuels-rs/blob/adf81bd451d7637ce0976363bd7784408430031a/packages/fuels-contract/src/parameters.rs#L15) and passing it to a chain method called `call_params`.
```

Comment:
Actually, we should point it to the actual docs, which will be up-to-date. I'd rather not use anchor here because it would be placing the anchor in a non-example/non-test piece of code. Point to [this](https://docs.rs/fuels/latest/fuels/core/parameters/struct.CallParameters.html#) should be fine.